### ISSUE TITLE
Error Level 1.12.2 to 1.13.2

### ISSUE THREAD

**Issue by TheCalypso** [mcMMO-Dev/mcMMO-Classic#11]
Hello,
I have a little problem with MCMMO of "then some time I spent my server in 1.13.2 but the levels of different talents have not registered and everything has been reset to 0. Is this normal or there is there a solution to this?

Thank you for your help.

**Comment by t00thpick1** [comment]
I'm sorry I'm having trouble understanding your issue, can you rephrase the question?

**Comment by TheCalypso** [comment]
All right, I switched my server from version 1.12.2 to 1.13.2. The players had different levels in fishing, mining etc ... in the passage of 1.13.2 they lost all their levels. Can you help me ?

**Comment by t00thpick1** [comment]
Interesting.  are you using MySQL or Flatfile?

**Comment by TheCalypso** [comment]
Flatfile
